website/static/js/profile.js

```diff
@@ -483,17 +483,34 @@ var SocialViewModel = function(urls, modes) {
     TrackedMixin.call(self);
 
     self.addons = ko.observableArray();
+    
+    self.profileWebsites = ko.observableArray();
+
+    self.profileWebsitesLength = ko.computed(function() {
```

Comment:
Can `self.profileWebsitesLength` be removed? I don't see it being used anywhere else.